CHANGELOG.md

```diff
@@ -5,6 +5,43 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 All notes should be prepended with the location of the change, e.g. `(proto)` or `(resolver)`.
 
+## 0.25.0
+
+0.25.0 represents a large release for the Hickory DNS project. Over 14 months since 0.24.0, we've
+added two new maintainers, divergentdave and marcus0x62, and have addressed many limitations.
+A team from Ferrous Systems [shored up our support for
+DNSSEC](https://ferrous-systems.com/blog/hickory-dns-client/), and we addressed a number
+of findings from our first security audit.
+
+### Breaking changes
```

Comment:
This list is not exhaustive. I think that's okay, but it should probably be mentioned. https://github.com/hickory-dns/hickory-dns/pull/2498 is an example of a breaking change, and even worse, it's a breaking change that doesn't break compilation in a lot of cases but will break config parsing f.ex. for people using the serde enum variant names.

Reply:
Yes, I'll add a note that this is not exhaustive. Do you want to contribute a similarly styled bullet point about your PR?

Reply:
I've added a bullet about this.